src/core/time.js

```diff
@@ -8,6 +8,14 @@
  * This syncs with Crafty's internal clock, and so should generally be preferred to using methods such as `setTimeout`.
  */
 module.exports = {
+    /**@
+     * #.delaySpeed
+     * @comp Delay
+     *
+     * The rate of the delay.  This property defaults to 1.
```

Comment:
Maybe add a clarifying comment, e.g. that a speed of 2 means the delay will complete in half the time?